### PR TITLE
Add e2e-aws-single-node-serial as a periodic job

### PR DESCRIPTION
**Pull request by omertuc**
Example run https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/16290/rehearse-16290-pull-ci-openshift-machine-config-operator-master-e2e-aws-single-node-serial/1410245257366343680

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml [hunk 1]
```diff
@@ -110,6 +110,14 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
     workflow: openshift-e2e-aws-single-node
+- as: e2e-aws-single-node-serial
+  cron: 0 0 * * *
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:114]:
```suggestion
  interval: 24h
```

Comment (omertuc) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:114]: Want to keep it consistent with the [other](https://github.com/openshift/release/blob/7cc0b2d767328e6667627d306bbce6dec381f10a/ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml#L106-L125) SNO jobs without modifying them (I'm afraid modifying them from 0 0 * * * to 24h might change the exact time they run daily, I don't know how it works)

Comment (omertuc) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:114]: Let me know if you still insist I change this :)

Comment (petr-muller) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:114]: I think that we switched the other jobs to `interval` today, too ;)

Comment (petr-muller) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:114]: https://github.com/openshift/release/pull/19866

Comment (omertuc) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml]: What are the odds... Fixed & force pushed

**Review by petr-muller — APPROVED** [review]

**Pull request merged**